### DISCUSSION TITLE
Duplicate button in entry types and possibility of dividing them into groups

### DISCUSSION THREAD

**Discussion by scoch5** [craftcms/cms#14726] [Ideas]
When creating EntryType, "duplicate" or "save as new element" as for entries would be very convenient.
Furthermore, entry types could be organized into groups, as is done for fields. This would help a lot in managing them better when there are many of them.

**Comment by terryupton** [comment]
I absolutely think the EntryType would benefit from a "duplicate" button, on the site I am currently building, this would have saved significant time if I could have duplicated the EntryTypes (show here labelled as blocks). I also agree that having the ability to group EntryTypes would have a vastly positive impact on the Development experience. 

Here I have tried to prepend the names to help create some grouping and clarity in the chaos and I am only one section into the build (ignoring what come over from Craft 4). 

<img width="1145" alt="Screenshot 2024-04-02 at 16 15 20" src="https://github.com/craftcms/cms/assets/5698596/8c2ca371-2521-4b49-9553-280c59ecc5fd">

My naming probably could do with a little clarity and inspiration too - but establishing what is a 'Page/Entry' vs a Block (Matrix field) or Partial (Matrix in Matrix) would definitely help keeping a handle and control on the potentially vast amount if EntryTypes.
The (C4) Craft 4 auto created (as part of the upgrade) entries have yet to be cleaned up and there is potentially a little duplication there, that might be served by a single EntryType bit not a huge amount.

**Comment by itsmattsoria** [comment]
Absolutely agree on both features! This would be a huge improvement. Once those entry types start stacking up, and they really do given how entry types are used now, it quickly becomes hard to sort though the pile.
